plugins/jira-dashboard-backend/config.d.ts

```diff
@@ -0,0 +1,34 @@
+/*
+ * Copyright 2023 The Backstage Authors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+export interface Config {
+  /** Configuration options for the Jira Dashboard plugin */
+  jira: {
```

Comment:
This is one that I'm not sure the right solution for but there should be some thinking on it. The Roadie plugin uses the same root level config. If someone is using both what would be the results of a config check? So the results of:`yarn backstage-cli config:check --config app-config.yaml --lax --strict`. I think it should be fine but one way to avoid this would be to use `jira-dashboard` instead. Again something to think about 🤔